src/engine/enginedelay.cpp

```diff
@@ -24,12 +24,12 @@
 const int kiMaxDelay = 40000; // 208 ms @ 96 kb/s
```

Comment:
We could do it if we just pick the maximum. If we use 80000 for the compensation delay and 40000 for all others, this should be a good deal for this PR. 400 ms should be more then enough to cover a 92 ms buffer: 92 in 92 out + 92 soundcard in + 92 + out =  368 ms